src/main/resources/tech/aroma/sql/queries/select_activity_event.sql

```diff
@@ -0,0 +1,8 @@
+------------------------------------------------------------------------------
+-- SELECTS AN EVENT FROM THE ACTIVITY TABLE
+------------------------------------------------------------------------------
+
+SELECT *
```

Comment:
Really, you only need the `serialized_event` field. Might wanna shrink this query.
![bitmoji](https://render.bitstrips.com/v2/cpanel/9954423-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)